routes/sites.js

```diff
@@ -90,6 +92,32 @@ async function getSites (req, res, next) {
     res.status(200).json({ siteNames })
 }
 
+/* Checks if a user has access to a repo. */
+async function checkHasAccess (req, res, next) {
+  try {
+    const { accessToken, userId } = req
+    const { siteName } = req.params
+    
+    const endpoint = `https://api.github.com/repos/${ISOMER_GITHUB_ORG_NAME}/${siteName}/collaborators/${userId}`
+    const resp = await axios.get(endpoint, {
+      headers: {
+        Authorization: `token ${accessToken}`,
+        "Content-Type": "application/json",
+      }
+    })
+    console.log(resp)
```

Comment:
Can we remove this log statement?

Reply:
Whoops thanks for catch that, resolved in https://github.com/isomerpages/isomercms-backend/pull/89/commits/ba92d4b3ca2f41848a714c49e6c0d1ee89dbfea5